client/src/components/AutoSuggestInput.js

```diff
@@ -16,20 +16,20 @@ class AutoSuggestInput extends Component {
     this.onChange = address => this.setState({ address, error: '' });
   }
 
-  handleSelect = address => {
+  handleSelect(address) {
```

Comment:
I think you will want to keep this as an arrow function since it uses `this`. I'm happy to talk more about that if you have questions